docs/source-app/workflows/ssh/index.rst

```diff
@@ -0,0 +1,124 @@
+
+#################################
+Debug cloud apps via SSH
+#################################
+
+**Audience:** Users that want to debug their cloud apps on their own machine.
+
+SSH allows you to access all your Lightning components in real time.
+This can be useful if you execute your app in a different environment than you develop it.
+This is often the case with applications that rely on GPUs.
+
+You can inspect the entire file system and monitor the runtime environment.
+
+----
+
+**********************************************************
+Add SSH key to lightning
+**********************************************************
+
+Before you can SSH to cloud machines, you will need to generate a new private SSH key, add it to the SSH agent, and add the public SSH key to your account on Lightning.
```

Comment:
Maybe mention here that the following three steps are one-time only? Or do you think it is clear to the user? Maybe it is

Reply:
Depends on the reader - I've added a sentence here in case an unexperienced user follows this to.